readme.md

```diff
@@ -456,6 +458,14 @@ test.cb(t => {
 });
 ```
 
+### Optional TAP output
+
+AVA can generate TAP output via `--tap` option for use with any 3rd-party reporters.
+
+```bash
+$ ava --tap | tap-spec
```

Comment:
> Should we include a screenshot or that'd be too much?

Sure! Here you go. Make sure to use `<img>` and half its width so it becomes retina.

![screen shot 2015-12-21 at 15 58 00](https://cloud.githubusercontent.com/assets/170270/11933560/df804f74-a7fc-11e5-9241-fab428c82f9a.png)